CoreFoundation/Base.subproj/CoreFoundation_Prefix.h

```diff
@@ -203,6 +206,7 @@ strlcat(char * dst, const char * src, size_t maxlen) {
     }
     return dstlen + srclen;
 }
+#endif
```

Comment:
Why is fd_mask defined in an else clause with definitions of strlcpy/strlcat? Are the two related in some way that I am missing?